src/day4/day_4.go

```diff
@@ -22,10 +29,40 @@ func calcPoints(winners int) int {
 	return int(math.Exp2(float64(winners - 1)))
 }
 
+// compares the number and winner arrays and returns the number of matching values
 func calcWinners(card card) int {
 	return len(intersect.Simple(card.winners, card.numbers))
 }
 
+// takes a look at the passed card, and based on how many are collected (always at least one) it then adds extra cards
+func calcCardCollection(card card) {
```

Comment:
I reckon we could have used a pointer here to avoid having to refer to the global `cards` variable